docs/tutorials/multimodal/object_detection/quick_start/quick_start_coco.md

```diff
@@ -163,6 +170,73 @@ better_predictor.evaluate(test_path)
 For how to set those hyperparameters and finetune the model with higher performance, 
 see :ref:`sec_automm_detection_high_ft_coco`.
 
+### Inference
+Now that we have gone through the model setup, finetuning, and evaluation, this section details the inference. 
+Specifically, we layout the steps for using the model to make predictions and visualize the results.
+
+To run inference **on the entire test set**, perform:
+
+```python .input
+pred = predictor.predict(test_path)
+print(pred)
+```
+The output `pred` is a `pandas` `DataFrame` that has two columns, `image` and `bboxes`, where
+- in `image`, each row contains the image path
+- in `bboxes`, each row is a list of dictionaries, each one representing a bounding box: 
+  - `{"class": <predicted_class_name>, "bbox": [x1, y1, x2, y2], "score": <confidence_score>}`
+
+Note that, by default, the `predictor.predict` does not save the detection results into a file.
+
+To run inference and save results, run the following:
+```python .input
+pred = better_predictor.predict(test_path, save_results=True)
+```
+Here, we save `pred` into a `.txt` file, which exactly follows the same layout as in `pred`.
+You can use a predictor initialzed in anyway (i.e. finetuned predictor, predictor with pretrained model, etc.).
+Here, we demonstrate using the `better_predictor` loaded previously. 
+
+You can also download an image and **run inference on that single image**. The follow is an example:
+
+Download the example image:
+```python .input
+from autogluon.multimodal import download
+image_url = "https://raw.githubusercontent.com/dmlc/web-data/master/gluoncv/detection/street_small.jpg"
+test_image = download(image_url)
+```
+
+Run inference:
+
+```python .input
+pred_test_image = better_predictor.predict({"image": [test_image]}, as_pandas=True)
+print(pred_test_image)
+```
+
+### Visualizing Results
+To run visualizations, ensure that you have `opencv` installed. If you haven't already, install `opencv` by running 
+```python .input
+!pip install opencv-python
+```
+
+To visualize the detection bounding boxes, run the following:
+```python .input
+from autogluon.multimodal.utils import visualize_detection
+
+conf_threshold = 0.4  # Specify a confidence threshold to filter out unwanted boxes
+visualization_result_dir = "./"  # Use the pwd as result dir to save the visualized image
+
+visualized = visualize_detection(
+    pred=pred[:1],
```

Comment:
I though we are visualizing the new `pred_test_image`, no?

Reply:
no we are visualizing `pred`, which is the output from the tiny motorbike dataset.